### DISCUSSION TITLE
How can i hire someone to do a white-label of the app?

### DISCUSSION THREAD

**Discussion by sirpantuflas** [RocketChat/Rocket.Chat.ReactNative#3444] [Q&A] [answered]
Requirements:
-Connect just android app to an owned server.
-Don't allow app to store cache, or erase it if there's no connection to server. Like using the "Clear local server cache" button behaviour.

**Comment by sirpantuflas** [comment]
Is there any way to hire someone for an apk of rocket chat with a few changes? I'm sorry for the inconvenience but I'm in a little hurry, wish you the best @sampaiodiego

**Comment by diegolmello** [comment, marked as answer]
We have a jobs channel for that https://open.rocket.chat/channel/jobs

**Reply by sirpantuflas** [reply]
Thank you very much, and sorry for the trouble.
